WordPressKit/WordPressKit/PluginDirectoryEntry.swift

```diff
@@ -152,15 +150,32 @@ extension PluginDirectoryEntry: Decodable {
 }
 
 // Since the WPOrg API returns `author` as a HTML string (or freeform text), we need to get ugly and parse out the important bits out of it ourselves.
+// Using the built-in NSAttributedString API for it is too slow — it's required to run on main thread and it calls out to WebKit APIs,
+// making the context switches excessively expensive when trying to display a list of plugins.
```

Comment:
The comment mentions the main thread, but this is still extracting on the main thread. I'm wondering if it's best to keep the extracted data as author/authorURL in the model, and just make sure `Endpoint.parseResponse(data:)` is called in a background thread.

Reply:
@koke excellent point! Updated, Is that more-or-less what you had in mind?